docs/development/FeatureFlag.md

```diff
@@ -0,0 +1,19 @@
+# Feature Flags - Policy and Intended Purposes for Sitrep Development
+
+## Why Feature Flags?
+We intend to implement and use Feature Flags to enable continuous development and testing of Sitrep in one branch.
+- Use case example: Development & testing of changes to existing features (e.g. a clock for the feed view of the Operation Journal).
+- Use case example: A specific feature is requested by an organization and they will beta-test it. Multiple features might be in development and tested by different users at the same time.
+- Use case example: A/B-Testing of UI-components or processes.
+
+
+## Sitrep Feature Flags Policy:
+1. Responsibility for each Feature Flag and adherence to this policy lies with the developer implementing it.
+2. Feature Flags shall only be implemented in front-end UI-elements, to enable or disable access to certain functionalities. 
+3. No Feature Flags must be implemented in back-end code. 
+4. A list of all prior and currently active Feature Flags shall be kept and include at least the following: intended Use Case, intended life-span, date implemented, date removed, developer name. 
```

Comment:
Where should that be kept? Furthermore, I would adjust developer name to "responsible person"

Reply:
Suggestion:
`4. A list of all prior and currently active Feature Flags shall be kept on Git _(insert reference here)_ and include at least the following information: intended Use Case, intended life-span, date implemented, date removed, @RedGecko/core-development member name. `